src/handlers/handlePreviousQuestion.js

```diff
@@ -2,15 +2,21 @@
 
 import showCurrentQuestion from "./showCurrentQuestion.js";
 import { quizData } from '../data.js';
+import getDOMElement from '../utils/getDOMElement.js';
+import { SUBMIT_BTN_ID } from '../constants.js';
+import {NEXT_QUESTION_BUTTON_ID } from '../constants.js';
 
-const  handlePreviousQuestion = () => {
-    if (quizData.currentQuestionIndex < 1) {
-        quizData.questions[0];
-      } else {
-        quizData.currentQuestionIndex = quizData.currentQuestionIndex - 1;
-      }
-
-    showCurrentQuestion();
+const handlePreviousQuestion = () => {
+    
+     if (quizData.currentQuestionIndex === 9) {
```

Comment:
Why nine?

Reply:
Nine because it should be in the object of the last question: quizData.questions[**9**];